src/main/java/com/Novi/TechItEasy/services/TelevisionService.java

```diff
@@ -145,6 +151,85 @@ public TelevisionDto changeTelevision(Long id, TelevisionInputDto television) {
     }
 
 
+    ///// For assigning remote controllers to television by id /////
+    public TelevisionDto assignRemoteControllerToTelevision(Long televisionId, IdInputDto remoteId) {
+        Optional<Television> fetchedTelevision =  televisionRepository.findById(televisionId);
+        Optional<RemoteController> fetchedRemoteController = remoteControllerRepository.findById(remoteId.id);
+
+        if (fetchedTelevision.isPresent()) {
+            if (fetchedRemoteController.isPresent()) {
+
+                fetchedTelevision.get().setRemoteController(fetchedRemoteController.get());
+
+                televisionRepository.save(fetchedTelevision.get());
```

Comment:
Ik betwijfel of deze `fetchedTelevision.get()` werkelijk de relatie met remotecontroller bevat, omdat het in een Optional staat. Anyways, het is niet nodig om 2 keer (en hier onder nog vaker) de Television uit de optional te halen met de get() methode